cpu/esp32/doc.txt

```diff
@@ -332,11 +332,11 @@ ESP-IDF, the official SDK from Espressif, can be downloaded and installed as GIT
 
 ```
 cd $HOME/esp
-git clone --recursive https://github.com/espressif/esp-idf.git
+git clone https://github.com/espressif/esp-idf.git
 cd esp-idf
-git checkout -q f198339ec09e90666150672884535802304d23ec
-cd components/esp32/lib
-git checkout -q 534a9b14101af90231d40a4f94924d67bc848d5f
+git checkout f198339ec09e90666150672884535802304d23ec
```

Comment:
Seems to work. Wouldn't it be better to checkout with `-q` option? The whole warning/information text might be misleading for the use.

Reply:
Different tastes I guess. At first I was wondering if `-q` was doing anything special since I usually don't use it.

But I also think it's good to see what's going on, no warnings afair.

Reply:
> Different tastes I guess.

Indeed. You know the UNIX principle. No messages are good messages.

> But I also think it's good to see what's going on, no warnings afair.

OK